lib/plugins/splitIOFeatureManagerPlugin.ts

```diff
@@ -0,0 +1,123 @@
+import * as path from 'path'
+import * as process from 'process'
+
+import { SplitFactory } from '@splitsoftware/splitio'
+import type SplitIO from '@splitsoftware/splitio/types/splitio'
+import type {
+  Attributes,
+  Properties,
+  SplitKey,
+  Treatment,
+  TreatmentWithConfig,
+} from '@splitsoftware/splitio/types/splitio'
+import type { FastifyInstance } from 'fastify'
+import fp from 'fastify-plugin'
+
+const DISABLED_TREATMENT: Treatment = 'control'
+
+declare module 'fastify' {
+  interface FastifyInstance {
+    splitIOFeatureManager: SplitIOFeatureManager
+  }
+}
+
+export interface SplitIOOptions {
+  isEnabled: boolean
+  apiKey: string
+  debugMode: boolean
```

Comment:
Looks like we use a mix of `debug` and `debugMode` naming.

Reply:
fixed